### PR TITLE
18769 - EFT switching logic fix

### PR DESCRIPTION
**Pull request by rodrigo-barraza**
Added all admins as notification recipients when switching to EFT payment method

*Issue #:*
https://github.com/bcgov/entity/issues/18769

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

File: auth-web/src/components/auth/account-settings/payment/AccountPaymentMethods.vue [hunk 1]
```diff
@@ -340,6 +345,26 @@ export default defineComponent({
           state.paymentMethodChanged = false
           initialize()
           orgStore.setCurrentOrganizationPaymentType(state.selectedPaymentMethod)
+          if (state.selectedPaymentMethod === PaymentTypes.EFT) {
+            const recipientList = []
+            await orgStore.syncActiveOrgMembers()
+            state.activeOrgMembers.forEach((member) => {
```
Comment (seeker25) [inline auth-web/src/components/auth/account-settings/payment/AccountPaymentMethods.vue:351]: should we be doing this for other payment methods as well? 

Also what happens if there are too many coordinators or admins?

**Review by seeker25 — APPROVED** [review]

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-account-dev--pr-3002-alnut2ud.web.app

File: auth-web/package-lock.json [hunk 2]
```diff
@@ -1,16 +1,13 @@
...
-  "version": "2.6.85",
+  "version": "2.6.86",
   "lockfileVersion": 2,
   "requires": true,
   "packages": {
     "": {
       "name": "auth-web",
-      "version": "2.6.85",
-      "dependencies": {
-        "@bcrs-shared-components/base-address": "2.0.3",
```
Comment (seeker25) [inline auth-web/package-lock.json:11]: Looks like a couple of these components dropped out

**Comment by seeker25** [comment]
deleted the comment, it contained people's personal email addresses..

**Pull request merged**